<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FACET</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
    <property name="centerButtons">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>551</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPre">
     <attribute name="title">
      <string>Pre-Processing</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>471</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Input Layers</string>
      </property>
      <widget class="QPushButton" name="browse_nhd_pre">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="browse_dem_pre">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_nhd_pre">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>19</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>DEM</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="Streamlines">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Streamlines</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_dem_pre">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <zorder>browse_nhd_pre</zorder>
      <zorder>browse_dem_pre</zorder>
      <zorder>txt_nhd_pre</zorder>
      <zorder>label_5</zorder>
      <zorder>Streamlines</zorder>
      <zorder>txt_dem_pre</zorder>
      <zorder>browse_streamlines_pre</zorder>
      <zorder>browse_hand_pre</zorder>
      <zorder>txt_hand_pre</zorder>
      <zorder>txt_streamlines_pre</zorder>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>471</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Output Layers</string>
      </property>
      <widget class="QPushButton" name="browse_streamlines_pre">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_hand_pre">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>HAND grid</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Streamlines</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_streamlines_pre">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="browse_hand_pre">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>471</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Paths</string>
      </property>
      <widget class="QLineEdit" name="txt_taudem_pre">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_whitebox_pre">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TauDEM Tools</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Whitebox Tools</string>
       </property>
      </widget>
      <widget class="QPushButton" name="browse_taudem_pre">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="browse_whitebox_pre">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QPushButton" name="browse_mpi_pre">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_mpi_pre">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>291</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>22</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MPI</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QCheckBox" name="chk_wg_pre">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>181</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Generate Stream Start Points Grid</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chk_whitebox_pre">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>201</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Run Whitebox BreachDepressions</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chk_taudem_pre">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Run TauDEM Tools</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tabXns">
     <attribute name="title">
      <string>Channel Metrics - Xns</string>
     </attribute>
     <widget class="QLineEdit" name="textVertIncrement">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>270</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkCreateXns">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Build and Write Cross Section File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBankPts">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkMetricsXns">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Channel Metrics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseXns">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>360</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DEM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textFitLength">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textStreams_Xns">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>360</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cross Sections</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseStreams_Xns">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>390</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bank Points</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textSlpThresh">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>330</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textRatioThresh">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textXnLength">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textDEM_xns">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Fit Length</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textXns">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>360</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cross Section Spacing</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textBankPts">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>390</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="browseDEM_Xns">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cross Section Length</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Reach ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Streamlines</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>270</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical Increment</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textXnSpacing">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>300</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ratio Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>330</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Slope Threshold</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxXns">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_fit_units">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_xngap_units">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_xnlength_units">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ivert_units">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCurvature">
     <attribute name="title">
      <string>Channel Metrics - Curvature</string>
     </attribute>
     <widget class="QPushButton" name="browseStreams_Curve">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textBankPixels">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textDEM_curvature">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxCurvature">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bank Pixels</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseDEM_Curve">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Reach ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Streamlines</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkBankPixels">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Delineate bank pixels</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DEM</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseBankPixels">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textStreams_curvature">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkPixelMetrics">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Channel Metrics</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>122</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Window Size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>87</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Window Step</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFP">
     <attribute name="title">
      <string>Floodplain Analysis</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>506</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblProgBar">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>483</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
